ocs_ci/ocs/platform_nodes.py

```diff
@@ -1674,10 +1674,32 @@ def stop_nodes(self, nodes):
             "Stop nodes functionality is not implemented"
         )
 
-    def start_nodes(self, nodes):
-        raise NotImplementedError(
-            "Start nodes functionality is not implemented"
-        )
+    def start_nodes(self, nodes, timeout=540, wait=True):
+        """Start Azure vm instances
+
+        Args:
+            nodes (list): The OCS objects of the nodes / Azure Vm instance
+            wait (bool): True for waiting the instances to start, False otherwise
+            timeout (int): time in seconds to wait for node to reach 'ready' state.
+        """
+        if not nodes:
+            logger.error("No nodes found to start")
+            raise ValueError
+        node_names = [n.name for n in nodes]
+        for node_name in node_names:
+            self.azure.start_az_vm_instance(node_name)
+        if wait:
+            """
+            When the node is reachable then 
+            the node reaches status Ready.
+            """
```

Comment:
please use `#` here:

```suggestion
            # When the node is reachable then the node reaches status Ready.
```

Reply:
Incorporated changes